### PR TITLE
Release/1.3.3

### LINKED ISSUE #69: Release version 1.3.3
### Describe your question

This issue is for tracking changes for the 1.3.3 release. Target release date: June 2024.

### [Release steps](https://github.com/10up/wp-newrelic/blob/develop/CONTRIBUTING.md#release-instructions)

- [x] Branch: Starting from `develop`, cut a release branch named `release/1.3.3` for your changes.
- [x] Version bump: Bump the version number in `newrelic-reporting-for-wordpress.php` and `readme.txt` if it does not already reflect the version being released.
- [x] Changelog: Add/update the changelog in `CHANGELOG.md` and `readme.txt`.
- [x] Props: update `CREDITS.md` file with any new contributors, confirm maintainers are accurate
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.gitattributes`.
- [x] Readme updates: Make any other readme changes as necessary. `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content. The two are slightly different.
- [x] Make a non-fast-forward merge from your release branch to `develop` (or merge the pull request), then do the same for `develop` into `trunk`, ensuring you pull the most recent changes into `develop` first (`git checkout develop && git pull origin develop && git checkout trunk && git merge --no-ff develop`). `trunk` contains the stable development version.
- [x] Push: Push your trunk branch to GitHub (e.g. `git push origin trunk`).
- [x] [Compare](https://github.com/10up/figma-block/compare/trunk...develop) trunk to develop to ensure no additional changes were missed.
- [x] Release: Create a [new release](https://github.com/10up/wp-newrelic/releases/new), naming the tag and the release with the new version number, and targeting the `trunk` branch. Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the closed issues on the [milestone](https://github.com/10up/wp-newrelic/milestone/3?closed=1).
- [x] SVN: Wait for the [GitHub Action](https://github.com/10up/wp-newrelic/actions) to finish deploying to the WordPress.org repository. If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [x] Check WordPress.org: Ensure that the changes are live on https://wordpress.org/plugins/wp-newrelic/. This may take a few minutes.
- [x] Close milestone: Edit the [milestone](https://github.com/10up/wp-newrelic/milestone/3) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description` field), then close the milestone.
- [x] Punt incomplete items: If any open issues or PRs which were milestoned for `1.3.3` do not make it into the release, update their milestone to `1.4.0` or `Future Release`.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by jeffpaul**
### Description of the Change
Release and version bump to 1.3.3.

Closes #69.

### How to test the Change
Verified manually via VS Code.

### Changelog Entry

n/a

### Credits

Props @jeffpaul.

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ## [Unreleased] - TBD
 
+## [1.3.3] - 2024-06-26
```
Comment (felipeelia) [inline CHANGELOG.md:7]: @jeffpaul do you mind adding the link we'll need at the bottom of the file? 

```
[1.3.3]: https://github.com/10up/wp-newrelic/compare/1.3.2...1.3.3
```

File: readme.txt [hunk 2]
```diff
@@ -113,32 +109,19 @@ If you identify any errors or have an idea for improving the plugin, please open
 
 == Changelog ==
 
+= 1.3.3 - 2024-06-26 =
+**Note this release bumps the PHP minimum from 7.3.11 to 8.0 and the WordPress minimum from 4.0 to 6.3.**
```
Comment (felipeelia) [inline readme.txt:113]: @jeffpaul I could be wrong but IIRC WP.org's parser will need an empty line before a bullet list to have it working properly. Feel free to disconsider if you know it doesn't.

**Review by felipeelia — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 2 above)
Comment (jeffpaul) [reply readme.txt:113]: Empty lines aren't needed, no.  See https://github.com/10up/simple-page-ordering/blob/develop/readme.txt && https://wordpress.org/plugins/simple-page-ordering/#developers as an example

File: CHANGELOG.md (hunk 1 above)
Comment (jeffpaul) [reply CHANGELOG.md:7]: Good catch, fixed in fa48f39.

**Review by felipeelia — APPROVED** [review]

**Pull request merged**